C/reverseNumber.c

```diff
@@ -2,7 +2,7 @@
 int main() {
```

Comment:
Can you add some documentation for this code in the form of comments? As the change you suggested is fine but I would like this code to be error-free from now. The content I require is what the program is all about and how the logic works here